Algorithm.md

```diff
@@ -0,0 +1,193 @@
+# Drawing Shapes with Computer Graphics Algorithms:
+
+> This file contains the codes for drawing various shapes using computer graphics algorithms:
+
+
+## 1. Digital Differential Analyzer (DDA) Line Code:
+```c
```

Comment:
Instead of c, write cpp after backticks.

Reply:
Yes!!